### DISCUSSION TITLE
What would the “accent” color be?

### DISCUSSION THREAD

**Discussion by markmacode** [EdenEast/nightfox.nvim#435] [Q&A] [answered]
I have created a port of this theme on Obsidian. I also created a template too for the extras dir on this repo (PR coming soon).

Obsidian has an option to select an accent color, which is used for all interactive buttons, links, and other areas on the app. I have this currently set to cyan just based on personal preference. But what would you say should nightfox's "accent" or "primary" color be for Obsidian or a web-based context?

**Comment by EdenEast** [comment, marked as answer]
I would say nightfox is primarily a blue-ish theme. When I have to pick a default color I usually pick `Cyan`.
